### PR TITLE
API-57: Clean the existant

### PR DESCRIPTION
**Pull request by skeleton**
- [x] Remove webServiceBundle
- [x] Remove firewall for rest API in the security.yml file
- [x] Create APIBundle

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | :negative_squared_cross_mark:
| Added Behats                      | :negative_squared_cross_mark:
| Changelog updated                 | :white_check_mark: 
| Review and 2 GTM                  | :clock1:
| Micro Demo to the PO (Story only) | :negative_squared_cross_mark:
| Migration script                  | :negative_squared_cross_mark:
| Tech Doc                          | :negative_squared_cross_mark:

:white_check_mark: Done and pass

:red_circle: Done but fail

:clock1: Done but pending

:negative_squared_cross_mark: Not needed

### CODE REVIEW

File: src/Pim/Bundle/APIBundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+    /**
+     * {@inheritDoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('pim_api');
+
+        // Here you should define the parameters that are allowed to
```
Comment (SamirBoulil) [inline src/Pim/Bundle/APIBundle/DependencyInjection/Configuration.php]: I'd remove this ?

File: src/Pim/Bundle/APIBundle/DependencyInjection/PimAPIExtension.php [hunk 2]
```diff
@@ -10,18 +10,18 @@
...
- * @copyright 2014 Akeneo SAS (http://www.akeneo.com)
- * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ * To learn more see {@link http://symfony.com/doc/current/cookbook/bundles/extension.html}
  */
-class PimWebServiceExtension extends Extension
+class PimAPIExtension extends Extension
 {
     /**
-     * {@inheritdoc}
+     * {@inheritDoc}
```
Comment (SamirBoulil) [inline src/Pim/Bundle/APIBundle/DependencyInjection/PimAPIExtension.php]: sure about this one ?

File: src/Pim/Bundle/APIBundle/DependencyInjection/PimAPIExtension.php [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace Pim\Bundle\WebServiceBundle\DependencyInjection;
+namespace Pim\Bundle\APIBundle\DependencyInjection;
```
Comment (a2xchip) [inline src/Pim/Bundle/APIBundle/DependencyInjection/PimAPIExtension.php]: Why do not we use ApiBundle standard naming form?

File: src/Pim/Bundle/APIBundle/DependencyInjection/PimAPIExtension.php (hunk 2 above)
Comment (skeleton) [inline src/Pim/Bundle/APIBundle/DependencyInjection/PimAPIExtension.php]: I changed it, thank (code was auto generated)

File: src/Pim/Bundle/APIBundle/DependencyInjection/Configuration.php (hunk 1 above)
Comment (skeleton) [inline src/Pim/Bundle/APIBundle/DependencyInjection/Configuration.php]: done

**Pull request merged**